puppet/modules/web/manifests/base.pp

```diff
@@ -9,9 +9,7 @@
     include web::letsencrypt
   }
 
-  class { 'apache':
-    default_vhost => false,
-  }
+  class { 'apache': }
```

Comment:
I'd prefer `include` since that doesn't conflict if there was already a `class { 'apache': }` somewhere.
```suggestion
  include apache
```